CMake/cdat_modules_extra/cdatmpi_configure_step.cmake.in

```diff
@@ -4,6 +4,7 @@ set(ENV{PKG_CONFIG} "@cdat_PKG_CONFIG_EXECUTABLE@")
 include(@cdat_CMAKE_BINARY_DIR@/cdat_common_environment.cmake)
 
 set(ENV{CC} mpicc)
+set(ENV{CXX} mpicxx)
 
```

Comment:
I don't think PROJ4 needs MPI or need to compile against the MPI compilers. We should look into the root cause. Since compiling against MPI adds specifically flags to the build.